### PR TITLE
Add GitHub PR template

### PR DESCRIPTION
**Pull request by JamesGuthrie**

### CODE REVIEW

**Review by bitwalker — APPROVED** [review]
LGTM. I'm a fan of keeping these simple, or people just end up ignoring them, but this seems like the sweet spot

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Description
+
+_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue._
+
+## Merge requirements
+
+Please take into account the following non-code changes that you may need to make with your PR:
+
+- [ ] CHANGELOG entry for user-facing changes
+- [ ] Updated the relevant documentation
```
Comment (VineethReddy02) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: I would also add a section to list issues fixed by the PR. Something like **Fixes**

**Review by VineethReddy02 — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (JamesGuthrie) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: The following is in the `Description` section:

> If it fixes a bug or resolves a feature request, be sure to link to that issue

**Review by VineethReddy02 — APPROVED** [review]

**Review by jgpruitt — APPROVED** [review]

**Review by cevian — APPROVED** [review]
LGTM

**Review by antekresic — APPROVED** [review]
This looks good to me. I like that its not too much for starters, we can always upgrade it later.

**Pull request merged**